src/backend/distributed/metadata/metadata_utility.c

```diff
@@ -1375,6 +1375,29 @@ ActiveShardPlacementList(uint64 shardId)
 }
 
 
+/*
+ * ShardPlacementListWithoutOldPlacements returns shard placements exluding
+ * the ones that are marked to be deleted.
+ */
+List *
+ShardPlacementListWithoutOldPlacements(uint64 shardId)
```

Comment:
Old is a pretty generic term. Thinking of something like LiveShardPlacementList / ExistingShardPlacementList.

or maybe Old->Deleted

Reply:
Orphaned is now used as the name in multiple places. So let's use that here too.